activation/activation.go

```diff
@@ -345,22 +280,12 @@ func (b *Builder) loop(ctx context.Context) {
 		ctx := log.WithNewSessionID(ctx)
 		err := b.PublishActivationTx(ctx)
 		if err == nil {
-			b.log.WithContext(ctx).With().Info("waiting for atx to propagate before building the next challenge",
-				log.Duration("wait", b.poetRetryInterval),
-			)
-			select {
-			case <-ctx.Done():
-				return
-			case <-time.After(b.poetRetryInterval):
-			}
 			continue
```

Comment:
I would not immediately retry again. We already had issues with nodes essentially DOS-ing on our poets and ending up in an endless loop without ever recovering.

Reply:
this case is for err == nil. if we exited successfully - node submitted atx in the epoch and will wait for the next round to submit for the next one